### PR TITLE
Created datastructures

### LINKED ISSUE #1: Data structures
Conditions met vector
Logical connector matrix
Preliminary unlocking vector
Preliminary unlocking matrix
Final unlocking vector

### PR DESCRIPTION
**Pull request by leddy231**
This creates the datastructures we need for different vectors, as well as parameters

resolves #1

### CODE REVIEW

File: app/src/main/java/decide/datastructures/PreliminaryUnlockingMatrix.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                boolean secondLIC = cmv.matrix[j];
+                boolean result = false;
+                switch (value) {
+                    case ANDD:
+                        result = firstLIC && secondLIC;
+                        break;
+                    case ORR:
+                        result = firstLIC || secondLIC;
+                        break;
+                    case NOTUSED:
```
Comment (Eyskha) [inline app/src/main/java/decide/datastructures/PreliminaryUnlockingMatrix.java:23]: When NOTUSED in LCM[i][j], the result should be true. The decision to launch or not do not depends on the correlation between CMV[i] and CMV[j].

**Review by Eyskha — CHANGES_REQUESTED** [review]
LGTM
PUM to adjust

File: app/src/main/java/decide/datastructures/PreliminaryUnlockingMatrix.java (hunk 1 above)
Comment (benronkth) [reply app/src/main/java/decide/datastructures/PreliminaryUnlockingMatrix.java:23]: I agree with Eyskha.

**Review by Eyskha — APPROVED** [review]

**Review by benronkth — APPROVED** [review]
Great job looks good

**Pull request merged**